gkr_iop/src/precompiles/bitwise_keccakf.rs

```diff
@@ -795,6 +795,11 @@ where
     E: ExtensionField,
 {
     type Trace = KeccakTrace<E>;
+
+    fn fixed_witness_group(&self) -> Vec<Vec<E::BaseField>> {
```

Comment:
fixed witness return type should be aligned with https://github.com/scroll-tech/ceno/blob/master/ceno_zkvm/src/tables/mod.rs#L31-L35 which is `RowMajorMatrix<E::BaseField>` to be commit into merkle tree